NEWS.md

```diff
@@ -1,6 +1,9 @@
 # CRE (Developing)
 
 ## Changed
+* Rename `interpret_select_rules` in `interpret_rules`.
```

Comment:
`interpret_select_rules` -> `interpret_rules` and also others.

Reply:
We never used this convention in the previous updates